requirements.in

```diff
@@ -7,8 +8,11 @@ connexion==2.13.0
 datadog==0.44.0
 flake8-bugbear==19.3.0
 flake8==3.7.7
+matplotlib==3.5.3
```

Comment:
I don't see where this is used?

Reply:
Ah it isn't, I was using it to visualize the stacks for debug purposes. Will be removed.